service.go

```diff
@@ -69,8 +69,8 @@ func NewConcreteService(
 		readiness:                    readiness,
 		retryBackoff: backoff.New(context.Background(), backoff.Config{
 			MinBackoff: 300 * time.Millisecond,
-			MaxBackoff: 600 * time.Millisecond,
-			MaxRetries: 50, // Sometimes the CI is slow ¯\_(ツ)_/¯
+			MaxBackoff: time.Second,
+			MaxRetries: 100, // Sometimes the CI is slow ¯\_(ツ)_/¯
```

Comment:
If I understand this correctly, this takes the timeout from 30s to 100s, which seems like a long time to be waiting for a service to start during a E2E test. Should we perhaps reduce this slightly?

Reply:
Sure. We can leave the `MaxBackoff` as it is now (0.6 seconds), which would give us a total of 60 seconds. Does that sound reasonable to you?

Reply:
Changed @ f4af71f2551105199fbdb1d81034421bc8d8a1ce

Reply:
Sounds good to me, thanks @ortuman!